src/fu-common-cab.c

```diff
@@ -15,58 +15,93 @@
 
 #include "fwupd-error.h"
 
-#ifdef HAVE_GCAB_1_0
+#ifndef HAVE_GCAB_1_0
```

Comment:
I wonder if as part of this PR it's time to bump to gcab 1.0 minimum?  Where is it missing still?  For the snap we can always just add another dependency while it builds to get around the 0.7 in Ubuntu 16.04.

Reply:
With my "I might have to rebase RHEL 7 hat on" I'd prefer if we can support this old version for a bit longer, although I concede I'm unlikely to chose this new branch. Won't the EPEL 7 builder fail with the old gcab version there?

Reply:
Yeah I guess it will.  Maybe the right time is when there is an EPEL8 or CentOS 8 builder that can be used then.